jax/_src/lax/linalg.py

```diff
@@ -1441,3 +1441,119 @@ def tridiagonal_solve(dl, d, du, b):
     raise ValueError(f'Only f32/f64 are supported, got {t}')
 
   return tridiagonal_solve_p.bind(dl, d, du, b, m=m, n=n, ldb=ldb, t=t)
+
+
+# Schur Decomposition
+
+
+def schur(x,
+          compute_schur_vectors=True,
+          sort_eig_vals=False,
+          select_callable=None):
+  return schur_p.bind(
+      x,
+      compute_schur_vectors=compute_schur_vectors,
+      sort_eig_vals=sort_eig_vals,
+      select_callable=select_callable)
+
+
+def _schur_impl(operand, *, compute_schur_vectors, sort_eig_vals,
+               select_callable):
```

Comment:
match indentation; here and below

Reply:
I'm sorry I couldn't figure out which two lines "here" and "below" referred to

Reply:
the indentation of the parameters on the line after the function definition should match the indentation of the parameters on the first line. i.e. this:
```python
def _schur_impl(operand, *, compute_schur_vectors, sort_eig_vals,
                select_callable):
```
rather than this:
```python
def _schur_impl(operand, *, compute_schur_vectors, sort_eig_vals,
               select_callable):
```
There are several other places where this comment applies in the code below this.

Reply:
Got it, thank you. I think the current commit fixes this.